packages/date-picker/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@kaizen/date-picker",
+  "version": "0.0.1",
+  "description": "Date Picker component",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '**/*.d.ts' '**/*.js' '**/*.map'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git",
+    "directory": "packages/date-picker"
+  },
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-design-system/issues"
+  },
+  "files": [
+    "**/*",
+    "!**/*.ts",
+    "!**/*.tsx",
+    "**/*.d.ts",
+    "!**/*.spec.*",
+    "!**/*.snap",
+    "!draft-templates",
+    "!stories",
+    "!docs",
+    "!tsconfig.dist.json"
+  ],
+  "author": "",
+  "private": false,
+  "license": "MIT",
+  "dependencies": {
+    "react-day-picker": "^7.4.9",
+    "classnames": "^2.3.1",
+    "@kaizen/draft-form": "^5.2.2",
+    "@kaizen/component-library": "12.2.0",
+    "@kaizen/deprecated-component-library-helpers": "^2.5.0",
```

Comment:
Having a quick squiz through and I don't think the `@kaizen/deprecated-component-library-helpers` helpers are used anywhere? Could possibly drop it from dependencies if not?

Reply:
Good catch! Not sure when that got introduced, will remove.